webpki-roots/Cargo.toml

```diff
@@ -16,6 +16,7 @@ hex = { workspace = true }
 percent-encoding = "2.3"
 rcgen = "0.11.1"
 ring = "0.17.0"
+rustls = "0.22"
```

Comment:
Please rebase to make no merge commit remains.

Reply:
I think now it is fixed as requested.